routes/root.js

```diff
@@ -218,26 +220,43 @@ route.get('/claims/view', (req, res) => {
     })
 })
 
-route.get('/claims/add', auth.ensureLoggedInGithub, (req, res) => {
-  res.render('pages/claims/add', {
-    menu: {
-      claims_add: 'active',
-      claims: 'claims-active'
-    }
-  })
-})
-
 route.get('/claims/:id', auth.adminOnly, (req, res) => {
   du.getClaimById(req.params.id)
     .then(claim => {
       if (!claim) throw new Error('No claim found')
-      res.render('pages/claims/id', { claim })
+      pullUrlDetail = getUrlDetails(claim["pullUrl"])
+      issueUrlDetail = getUrlDetails(claim["issueUrl"])
+      if(pullUrlDetail.type === "pull") {
+        du.getConflictedClaims(claim,issueUrlDetail)
+          .then(conflictedClaims => {
+            if(!conflictedClaims)
+              res.render('pages/claims/id',{claim, hasConflict: false })
+            else
+              res.render('pages/claims/id',{ claim, hasConflict: true, conflictedClaims })
+          })
+          .catch(err => {
+            res.send('Error getting conflicting claims')
+          })
+      }
+      else {
```

Comment:
There can be conflicts in the case when the pull url points to an issue as well. Handle that here.

Reply:
it means same person claimed issue more than once or someone else is claiming issue created by someone. right?

Reply:
will fix it.